### PR TITLE
fix CompileServiceSpec

### PR DESCRIPTION
**Pull request by danielkarch**

### CODE REVIEW

File: core/src/test/scala/org/http4s/rho/CompileServiceSpec.scala [hunk 1]
```diff
@@ -7,13 +7,12 @@ import org.http4s.{Method, Request, Uri}
 import org.specs2.mutable.Specification
 
 class CompileServiceSpec extends Specification {
-  import CompileService.Implicit.compiler
```
Comment (danielkarch) [inline core/src/test/scala/org/http4s/rho/CompileServiceSpec.scala:10]: Some tests need the stateful `CompileService` provided by `ServiceBuilder`.

File: core/src/test/scala/org/http4s/rho/CompileServiceSpec.scala [hunk 2]
```diff
@@ -22,35 +21,36 @@ class CompileServiceSpec extends Specification {
       val c = ServiceBuilder[IO]()
       getFoo(c)
 
-      "GetFoo" === new RRunner(c.toService()).checkOk(Request(uri=Uri(path="/hello")))
+      "GetFoo" === RRunner(c.toService()).checkOk(Request(uri=Uri(path="/hello")))
```
Comment (jcranky) [inline core/src/test/scala/org/http4s/rho/CompileServiceSpec.scala:24]: Removing the `new` fixed something? Or the fix was basically the import?

**Comment by danielkarch** [comment]
The fix was the import and removing the generic parameter from `getFoo` and `putFoo`. I removed the `new` because `RRunner` is a case class.

**Comment by jcranky** [comment]
ok, thanks :)

**Pull request merged**